ansible_collections/arista/avd/roles/eos_designs/docs/tables/new-network-services-bgp-vrf-config.md

```diff
@@ -0,0 +1,23 @@
+<!--
```

Comment:
Remember to include this file in the markdown file. Not sure if this should be role config or not...

Reply:
indeed